### PR TITLE
[metadata.themoviedb.org]: option to include adult movies in title searches

### PR DESCRIPTION
**Pull request by Oblomov**
Default to false, preserving old behaviour.

### CODE REVIEW

**Comment by Jalle19** [comment]
I'd drop the Italian translation, like @uNiversaI said we sync these from Transifex anyway.

File: addons/metadata.themoviedb.org/resources/language/English/strings.po [hunk 1]
```diff
@@ -43,3 +43,7 @@ msgstr ""
 msgctxt "#30006"
 msgid "Preferred Certification Country"
 msgstr ""
+
+msgctxt "#30007"
+msgid "Search for adult movies too"
```
Comment (xhaggi) [inline addons/metadata.themoviedb.org/resources/language/English/strings.po]: I don't like the wording, should we change it to something like "Include adult movies"

**Comment by Oblomov** [comment]
Thanks for the suggestions. I've pushed a new version with a slightly different wording and dropping the Italian translation

**Comment by xhaggi** [comment]
Thanks.

**Comment by MartijnKaijser** [comment]
@olympia ping

**Comment by olympia** [comment]
This is violating our no-porn policy. Even this is made optional, having it enabled could end up with accidental and non-wanted xxx artwork (think about a false-positive hits on automatic scrape). Don't think we want this?!

**Comment by MartijnKaijser** [comment]
This is indeed a case that is touching or no-porn policy. Flipping this switch on when adding sources is way too easy and reverting that is a nightmare

**Comment by mkortstiege** [comment]
Agreed. Stock scrapers should not include that and apply to our no-porn policy.

**Pull request closed without merging**